superset-frontend/package.json

```diff
@@ -183,13 +183,13 @@
     "react-reverse-portal": "^2.1.1",
     "react-router-dom": "^5.1.2",
     "react-search-input": "^0.11.3",
-    "react-select": "^3.1.0",
-    "react-sortable-hoc": "^1.11.0",
+    "react-select": "^3.2.0",
+    "react-sortable-hoc": "^2.0.0",
```

Comment:
no breaking changes here?

Reply:
seems so, though it's no longer maintained, for v18 we'll need a replacement

Reply:
yes there is no breaking change for react-sortable-hoc v2 and only works up to react v17. We will need migrate from react-sortable-hoc to @dnd-kit/sortable for react v18 https://github.com/clauderic/react-sortable-hoc#readme